post_office/management/commands/load_email_template.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from ...models import EmailTemplate
+from ...logutils import setup_loghandlers
+
+logger = setup_loghandlers()
+
+
+def _get_template(template_name):
+    """Find the template and return the raw template file content"""
+
+    template = loader.get_template(template_name)
+    content = template.template.source
+    content = content.strip()
```

Comment:
I don't think we should call `.strip()`, it could be that the user actually wants to have whitespaces before and after the content, this may affect how `content.txt` or `subject.txt` are displayed on the email.